### ISSUE TITLE
Demo Feedback: Move notification settings to user profile

### ISSUE THREAD

**Issue by Sephsekla** [WordPress/wp-feature-notifications#86]
The settings page for notifications had a very positive response. However it was suggested that it would make more sense to have these settings stored on a per-user level (maybe as user meta?), and that therefore this should be moved to the user settings.

A user should be able to manage their own notifications settings, and an administrator/someone with a specific role should be able to change this for others.

Maybe `edit_users` (`manage_network_users` for managing certain aspects on multisite), or potentially a new `manage_user_notifications` role?

**Comment by bacoords** [comment]
This is a great point. The Edit Profile/Edit User screen makes more sense. The "Settings" menu items are clearly for global/sitewide settings, (but I will say the Profile/User screen is getting a little long and jammed full of disparate settings).

**Comment by Sephsekla** [comment]
@bacoords not sure if that's a problem unique to this project to be honest, although maybe it's something we could look at solutions for that could be merged into core earlier?

**Comment by bacoords** [comment]
@Sephsekla Agreed - not really a problem for this project and a little off-topic. I guess the options are:

1. just add these settings as another chunk to the profile.php file
1. add to the profile.php AND user-edit.php (meaning admins could edit Notifications settings for other users)
1. keep the current implementation (of having Notifications under Settings- but they would only apply to the logged in user and we'd need to extend this to Subscriber level users for things like comment notifications) - this seems like the weirdest?
1. move the current implantation to a new screen under Users after the 'Profile' link
1. It's ONLY accessible via the link in the notifications hub and doesn't show up on the sidebar at all (similar to like `about.php`)

Just some thoughts about this. (This may also be better as a 'Discussion' than Issue until a path forward is chosen?)

**Comment by johnhooks** [comment]
@bacoords I'm curious are the options under Settings > Writing on an individual user basis? Looking through the options we have, I agree, nothing feels quite right. I lean towards your option 3, though if there isn't precedence for individual users settings in under the settings menu it would be odd. 

After looking into it, the writing settings seem to be site wide... which is odd. And all the standard settings menus are only available to users with 'manage_options' permissions, which is Admin and SuperAdmin in a Typical WordPress setup.

Edit: 

> move the current implantation to a new screen under Users after the 'Profile' link

I like option 4 the best. It needs to be on its own screen there will be a lot of detail, not on the profile page.

**Comment by bacoords** [comment]
The only screen currently that lets you change user-specific settings is the user profile edit screen. But even that opens another question: 

Can an administrator change the notification settings for another user (the way they can change all of the other settings)? If not, then the profile.php or a custom screen makes the most sense. If so, then profile.php AND user-edit.php makes the most sense.

**Comment by johnhooks** [comment]
@bacoords in my mind there could be system wide preferences. that could be overridden on a per user basis. 

Example:

Admin configures for all post update notifications to be send through an integration with slack to notify everyone of a change (this is global so it overrides everyones preference, unless they have made their own), This setting would be visible to the individual users, and allow them to override it, because they would prefer an email. It still gets sent to a slack channel, but also sends out an email to the specific user.

**Comment by EdithAllison** [comment]
Not sure if the Figma design is based on general or user level options? Design is tracked in https://github.com/WordPress/wp-feature-notifications/issues/357. May need changes to the design to allow for two level of settings.
